sprout/sprout/settings.py

```diff
@@ -166,7 +166,7 @@
 
     'free-appliance-shepherd': {
         'task': 'appliances.tasks.free_appliance_shepherd',
-        'schedule': timedelta(minutes=5),
+        'schedule': timedelta(minutes=1),
```

Comment:
Yes it is, since I made shepherd to only do max 1 appliance per group per run to balance it better. Therefore the reduction makes sense.